src/prefect/runner/submit.py

```diff
@@ -0,0 +1,250 @@
+import asyncio
+import inspect
+import uuid
+from typing import Any, Dict, List, Optional, Union, overload
+
+import anyio
+import httpx
+from typing_extensions import Literal
+
+from prefect.client.orchestration import get_client
+from prefect.client.schemas.filters import FlowRunFilter, TaskRunFilter
+from prefect.client.schemas.objects import FlowRun
+from prefect.context import FlowRunContext
+from prefect.flows import Flow
+from prefect.logging import get_logger
+from prefect.settings import (
+    PREFECT_EXPERIMENTAL_ENABLE_EXTRA_RUNNER_ENDPOINTS,
+    PREFECT_RUNNER_PROCESS_LIMIT,
+    PREFECT_RUNNER_SERVER_HOST,
+    PREFECT_RUNNER_SERVER_PORT,
+)
+from prefect.states import Pending
+from prefect.tasks import Task
+from prefect.utilities.asyncutils import sync_compatible
+
+logger = get_logger("webserver")
+
+
+async def get_current_run_count() -> int:
```

Comment:
Maybe we don't need this after all? The webserver API is going to check the runner to make sure it can accept work. I have a preference for keeping validation like that behind the server API.

Reply:
agreed, removed in [c42d5cc](https://github.com/PrefectHQ/prefect/pull/11476/commits/c42d5cc47e60d666d9e744cd6b9af9f70934a834)